incubator/virtualcluster/config/default/kustomization.yaml

```diff
@@ -28,7 +28,7 @@ resources:
 - ../rbac/auth_proxy_role.yaml
 - ../rbac/auth_proxy_role_binding.yaml
 
-patches:
+patchesStrategicMerge:
```

Comment:
Let's not make changes across _both_ HNC and non-HNC code. Can you please revert the changes to `virtualcluster`?

Reply:
Ah, sure. You right, I changed all the locations that had `patches` and forgot about the virtualcluster files. My bad! I'll change this.